### ISSUE TITLE
HTTP Task remains in scheduled state 

### ISSUE THREAD

**Issue by autodidactic** [Netflix/conductor#3566]
The workflow created from conductor UI using new workflow definition button creates a basic workflow with default HTTP task called get_population_data
The workflow remains in running state and http task is scheduled. I expect it to go to complete while skipping running , almost immiediately. but not the behaviour i see in 3.13.5

it remains in scheduled state

here is the json

{
"createTime": 1680633455822,
"name": "Conduc_3.13.5",
"description": "conductor workflow for 3.13.5",
"version": 1,
"tasks": [
{
"name": "get_population_data",
"taskReferenceName": "get_population_data",
"inputParameters": {
"http_request": {
"uri": "https://datausa.io/api/data?drilldowns=Nation&measures=Population",
"method": "GET"
}
},
"type": "HTTP",
"startDelay": 0,
"optional": false,
"asyncComplete": false
}
],
"inputParameters": [],
"outputParameters": {
"data": "${get_population_data.output.response.body.data}",
"source": "${get_population_data.output.response.body.source}"
},
"schemaVersion": 2,
"restartable": true,
"workflowStatusListenerEnabled": false,
"ownerEmail": "[example@email.com](mailto:example@email.com)",
"timeoutPolicy": "ALERT_ONLY",
"timeoutSeconds": 0,
"variables": {},
"inputTemplate": {}
}

steps to reproduce:

connect to conductor ui
click on definition
click blue button "new workflow definition"
enter name or workflow and desc and other mandatory fields
save it 

click on workbench and select the newly created workflow 
execute workflow
navigate to executions screen and wait for workflow to show up 
verify status is running and task is scheduled

any help or update would help us decide on how to proceed on this version or wait for fix

**Comment by jkaipa** [comment]
This is a long standing open issue. 
https://github.com/Netflix/conductor/issues/1153#issue-446775716

**Comment by ankit-gautam23** [comment]
Is it still open?

**Comment by manan164** [comment]
Hi @autodidactic , Do you see this behavior in the latest conductor?

**Comment by autodidactic** [comment]
yes i do see it in 3.13.5 version  we adopted this one since we had issues with dynamic fork join in prev version of server  which was 3.11.3 
we continue to face challenges in each version but this one seems to be covering most freq used test cases. thats why we skipped 3.13.3 and procured latest and greatest. Also there is a lack of precise indication on ui as to how http task would execute

**Comment by autodidactic** [comment]
Any update on if we can wait for 3.13.7 for a fix  in this issue

**Comment by autodidactic** [comment]
Is there documentation about it ?Will go ahead and try thatSent from my iPhone 13 PRO maxOn Jun 6, 2023, at 1:11 PM, narayanapadmanabhuni ***@***.***> wrote:﻿
did you added com.netflix.conductor:conductor-http-task dependency in the project. I see the http-task has been separated from conductor-core and shipped as a different module. When I added the http-task dependency it started working.

**Comment by narayanapadmanabhuni** [comment]
if you look at the conductor code, you can see that difference. earlier HttpTask is part of core module, now they moved it to httpTask sub-module/project. Thats how I figured it out. May be you can try
